Cargo.toml

```diff
@@ -10,15 +10,15 @@ edition = "2018"
 
 [dependencies]
 auto_impl = "1.0"
-ethereum = { git = "https://github.com/rust-ethereum/ethereum.git", rev = "3be0d8fd4c2ad1ba216b69ef65b9382612efc8ba", default-features = false }
+ethereum = { git = "https://github.com/rust-ethereum/ethereum.git", rev = "bbb544622208ef6e9890a2dbc224248f6dd13318", default-features = false }
 log = { version = "0.4", default-features = false }
 primitive-types = { version = "0.13", default-features = false, features = ["rlp"] }
 rlp = { version = "0.6", default-features = false }
 sha3 = { version = "0.10", default-features = false }
 
 # Optional dependencies
 environmental = { version = "1.1.2", default-features = false, optional = true }
-scale-codec = { package = "parity-scale-codec", version = "3.2", default-features = false, features = ["derive"], optional = true }
+scale-codec = { package = "parity-scale-codec", git = "https://github.com/paritytech/parity-scale-codec", rev = "334b99c9cc8def7408bcd91d1ff7bfd79f7e2915", default-features = false, features = ["derive"], optional = true }
```

Comment:
Asked for a new release on this package. To avoid conflicts on dependent projects, lets wait for a new release from parity side.

Reply:
It has been released: https://crates.io/crates/parity-scale-codec/3.7.5